app/code/Magento/Bundle/Test/Mftf/Test/BundleProductWithTierPriceInCartTest.xml

```diff
@@ -34,7 +34,7 @@
             <actionGroup ref="AdminDeleteCustomerActionGroup" stepKey="deleteCustomer">
                 <argument name="customerEmail" value="CustomerEntityOne.email"/>
             </actionGroup>
-            <amOnPage url="{{AdminLogoutPage.url}}" stepKey="logout"/>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="amOnLogoutPage"/>
```

Comment:
If you can keep the original `stepKey` that would be appreciated, as some developers might have attached their Actions to it.

Reply:
**Done**